traffic_generator_trex.py

```diff
@@ -53,7 +53,7 @@
     "./trex_stl_lib/external_libs/"))
 
 from trex_stl_lib.api import STLClient, STLError, STLPktBuilder, STLStream, \
-    STLTXCont, STLVmFixIpv4, STLVmFlowVar, STLVmWrFlowVar
+                      STLTXCont, STLVmFixIpv4, STLVmFlowVar, STLVmWrFlowVar
```

Comment:
Guess formatting should be like this to keep flake8 happy:

from trex_stl_lib.api import STLClient, STLError, STLPktBuilder, STLStream, \
                             STLTXCont, STLVmFixIpv4, STLVmFlowVar, \
                             STLVmWrFlowVar